varnish/varnish3.vcl

```diff
@@ -0,0 +1,41 @@
+sub vcl_recv {
+
+    # break early for OPTIONS requests
+    if (req.request == "OPTIONS") {
+        error 204 "No Content";
```

Comment:
They will be added in the `vcl_deliver` block, this is just to stop processing and not send the request to the origin